### PR TITLE
ufw: extend integration tests

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Extends integration tests for ufw. Prepares check mode tests for #49948 (which will fix the check mode bugs).

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
ufw

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @jbarotin

File: test/integration/targets/ufw/tasks/tests/basic.yml [hunk 1]
```diff
@@ -25,14 +44,30 @@
...
+- name: ipv4 allow (idempotency, check mode)
+  ufw:
+    rule: allow
+    port: 23
+    to_ip: 0.0.0.0
+  check_mode: yes
+  register: ipv4_allow_idem_check
 - assert:
     that:
+    # FIXME - ipv4_allow_check is changed
```
Comment (gundalow) [inline test/integration/targets/ufw/tasks/tests/basic.yml:57]: Is there a bug for this?

Comment (felixfontein) [reply test/integration/targets/ufw/tasks/tests/basic.yml:57]: Yep, there's a bug (#45604) and a PR (#49948) to fix this.

Comment (felixfontein) [reply test/integration/targets/ufw/tasks/tests/basic.yml:57]: (I mentioned the PR in the commit message of 5566158fec0702989303b74ea04dc4085def60a1, but not in the first line, so it is hard to find :) )

**Comment by felixfontein** [comment]
@gundalow thanks for reviewing and merging!

**Pull request merged**